connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -590,6 +590,16 @@ class SparkSession private[sql] (
   @Experimental
   def addArtifact(uri: URI): Unit = client.addArtifact(uri)
 
+  /**
+   * Add a single in-memory artifact to the client session with a specified target URI.
+   *
+   * Currently it supports files with extensions .jar and .class.
```

Comment:
In API documentation, we should better remove things like "Currently".  Should say sth like "Supported file extensions are `.jar` and `.class`".